app/build.gradle

```diff
@@ -59,17 +59,20 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
+    namespace 'com.example.marsphotos'
```

Comment:
What is the effect of this line?

Reply:
We are moving the package name from [app/src/main/AndroidManifest.xml](https://github.com/google-developer-training/basic-android-kotlin-compose-training-mars-photos/pull/1/files#diff-7fa6aef292187a049f7a4d6060d8df3ba212d838789c78940bd363344b1c38cd) to [app/build.gradle](https://github.com/google-developer-training/basic-android-kotlin-compose-training-mars-photos/pull/1/files/b15b3d86ea221b4717a2279bb77bd4fa7bad9f4d#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9)